src/java.base/share/classes/sun/security/ssl/SSLConfiguration.java

```diff
@@ -191,6 +196,26 @@ final class SSLConfiguration implements Cloneable {
         } else {
             maxInboundServerCertChainLen = inboundServerLen;
         }
+
+        Integer nstServerCount = GetIntegerAction.privilegedGetProperty(
+            "jdk.tls.server.newSessionTicketCount");
+        if (nstServerCount == null || nstServerCount < 1 ||
```

Comment:
Please permit zero here. It is useful for preventing the connection reset on connections where the client is only writing and never reading from the socket.

Reply:
I have been on the fence with zero.  I can support it.